guide/01-getting-started/install-and-set-up.ipynb

```diff
@@ -4,234 +4,100 @@
    "cell_type": "markdown",
```

Comment:
<p>Replace this section with:</p><p> </p><h2>Next Steps</h2><h2><br></h2><p><a href="https://developers.arcgis.com/python/guide/overview-of-the-arcgis-api-for-python/" target="_blank">This overview</a> describes how to use ArcGIS API for Python to write Python code, incorporating capabilities such as mapping, query, analysis, geocoding, routing, portal administration, and more. Browse the <a href="https://developers.arcgis.com/python/sample-notebooks/" target="_blank">sample notebooks</a> to get started. You may choose to experience these sample notebooks in a live environment as they are available as ArcGIS Notebooks.  </p><h2><br></h2>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/764/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>